### PR TITLE
allow '*' versions to match all

### PR DESCRIPTION
**Pull request by stefanpenner**

### CODE REVIEW

**Comment by stefanpenner** [comment]
@quaertym r?

Also, I would love to help you maintain this project. If you are interested let me know :)

File: tests/unit/dependency-checker-bower-test.js [hunk 1]
```diff
@@ -92,5 +92,10 @@ describe('EmberCLIDependencyChecker', function() {
       var project = createProject({ 'ember': '~/projects/emberjs' });
       assertNoBowerError(project);
     });
+
+    it('does error with a * dependency', function() {
```
Comment (rwjblue) [inline tests/unit/dependency-checker-bower-test.js]: Should be "does *not* error"

File: tests/unit/dependency-checker-npm-test.js [hunk 2]
```diff
@@ -82,5 +82,10 @@ describe('EmberCLIDependencyChecker', function() {
       var project = createProject({ 'ember-cli': '~/projects/ember-cli' });
       assertNoNpmError(project);
     });
+
+    it('does error with a * dependency', function() {
```
Comment (rwjblue) [inline tests/unit/dependency-checker-npm-test.js]: Should be "does *not* error.

**Comment by stefanpenner** [comment]
Lame. Copy and paste. Will fix when I get home

**Comment by stefanpenner** [comment]
fixed.

**Comment by quaertym** [comment]
@stefanpenner 
> Also, I would love to help you maintain this project. If you are interested let me know :)

That would be great! :beers:

**Comment by stefanpenner** [comment]
feel free to add me/rob to git + npm, so we can release and merge if you are busy.

**Pull request merged**